newIDE/app/src/ResourcesEditor/ResourcePropertiesEditor/index.js

```diff
@@ -21,6 +28,9 @@ type Props = {|
   project: gdProject,
   resourcesLoader: typeof ResourcesLoader,
   resources: Array<gdResource>,
+  onUpdateProperties: () => void,
```

Comment:
@blurymind Not caught this earlier, can we rename this `onResourcePathUpdated`?

Using the past tense for the verb because the action of update is already done, it's not something that must be done by the parent. Also Properties is too vague/misleading here, because if I change another property, it won't be called.